packages/line/src/LineSlicesItem.js

```diff
@@ -25,41 +25,45 @@ Chip.propTypes = {
     color: PropTypes.string.isRequired,
 }
 
-const LineSlicesItem = ({ slice, height, showTooltip, hideTooltip, isHover }) => (
-    <g transform={`translate(${slice.x}, 0)`}>
-        {isHover && (
-            <line
-                x1={0}
-                x2={0}
-                y1={0}
-                y2={height}
-                stroke="#000"
-                strokeOpacity={0.35}
-                strokeWidth={1}
+const LineSlicesItem = ({ slice, height, width, showTooltip, hideTooltip, isHover, xOffset }) => {
```

Comment:
Is this the correct order would you rather have xOffset next to height and width?